src/utils/colors.js

```diff
@@ -159,6 +159,15 @@ function getRgb(color) {
     };
   }
 
+  const hex3Regex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i.exec(color);
+  if (hex3Regex != null) {
+    return {
+      r: parseInt(hex3Regex[1], 16) * 17,
```

Comment:
this logic feels a bit unclear as a reader. it might be cleaner to simply `parseInt(hex3Regex[1] + hex3Regex[1], 16)` and so on

Reply:
Sure, let's do that. I agree that it's a bit unclear. The `* 17` seems wrong even though I know it's right. haha